### PR TITLE
Add changes for v2.20 for pubcloud

### PR DESCRIPTION
**Pull request by jesusbv**
## Description

Update the changes for 2.20 release

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [X] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [ ] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Review

Please check out our [review guidelines](https://github.com/SUSE/scc-docs/blob/master/team/workflow/code_review.md) 
and get in touch with the author to get a shared understanding of the change.

### CODE REVIEW

File: package/obs/rmt-server.changes [hunk 1]
```diff
@@ -8,6 +8,9 @@ Wed Oct 30 09:01:32 UTC 2024 - Natnael Getahun <natnael.getahun@suse.com>
   * rmt-server-pubcloud:
     * Fix LTSS product verification (bsc#1230154)
     * Fix activations check when no product info is available (bsc#1230157)
+    * Fix Azure SCC connection (bsc#1233314)
+    * Deny access to Azure Basic type images to LTSS
```
Comment (digitaltom) [inline package/obs/rmt-server.changes:12]:
```suggestion
    * Deny access of Azure Basic type images to LTSS
```

File: package/obs/rmt-server.changes [hunk 2]
```diff
@@ -8,6 +8,9 @@ Wed Oct 30 09:01:32 UTC 2024 - Natnael Getahun <natnael.getahun@suse.com>
...
+    * Allow SLE Micro system to access free SLES repositories (bsc#1230419)
```
Comment (digitaltom) [inline package/obs/rmt-server.changes:13]:
```suggestion
    * Allow SLE Micro system to access SLES repositories (bsc#1230419)
```

**Review by digitaltom — APPROVED** [review]
Please see comments

**Review by rjschwei — APPROVED** [review]
Suggestions should be included before merge

**Pull request merged**